hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -782,16 +930,28 @@ set_output_delay -min ${SPI_HOST_OUT_DEL_MIN} \
 set_output_delay -max ${SPI_HOST_OUT_DEL_MAX} \
     [get_ports "SPI_HOST_CS_L ${SPI_HOST_DATA_PORTS}"] \
     -clock SPI_HOST_SLOW_PASS_CLK -add_delay
-
+} else {
```

Comment:
The `else` case (and the one above on L909) do not have output_delay constraints. All other `$spec_constr` if/else blocks duplicate the commands within each case. Is that intentional?

Reply:
Yes, it actually is intentional. The PD provided feedback on this. The `spec_constr == 1` case in these two particular cases verifies the fast passthrough mode where the direct timing paths from an upstream host to a downstream Flash device is checked. The SPI Device is then defined as input and the SPI Host as output.

In contrast, the `spec_constr == 0` case verifies the slow passthrough mode where the direct timing paths from a downstream Flash to an upstream host are checked. Then, the SPI device is defined as output and the SPI Host is defined as input. In this case, SPI Device gets no input delay constraints and SPI Host not output delay constraints.

I'll document this in the SDC.

Reply:
Ah, and the PD team also confirmed that both constraint sets got verified for the tapeout.